### PR TITLE
Standardize the query of the APIs

### LINKED ISSUE #370: Alterar a consulta da API para o geocodigo 3304557 (Rio de Janeiro)
Atualmente a cidade do Rio de Janeiro (3304557) é a unica que resulta em consulta diferente na API.  Isso ocorre porque a tabela consultada para essa cidade é especifica, sendo uma exceção a todo o infodengue.

O objetivo da issue é uniformizar a consulta das APIs, tornando-a equivalente em todas as outras cidades (mesmas variaveis, mesmo dicionario de dados). Para isso, é suficiente apontar a consulta do Rio de Janeiro para as mesmas tabelas dos outros municipios, isto é,  historico_alerta, historico_alerta_chik e historico_alerta_zika.

### PR DESCRIPTION
**Pull request by eltonn**
Resolves #370 

Foi modificado o comportamento para utilizar a tabela historico_alerta ao invés de mrj.

### CODE REVIEW

**Comment by xmnlab** [comment]
```
Run git clone https://github.com/esloch/Data.git docker/dev_dumps
Cloning into 'docker/dev_dumps'...
Downloading alertademo_data.sql (612 MB)
Error downloading object: alertademo_data.sql (41ff70b): Smudge error: Error downloading alertademo_data.sql (41ff70bf7725aead03246d7d24d05334a7a463c6e546d360571fd03b9ceeccbb): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

Errors logged to /home/runner/work/AlertaDengue/AlertaDengue/docker/dev_dumps/.git/lfs/logs/20201210T182041.069933696.log
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: alertademo_data.sql: smudge filter lfs failed
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

```


it would be good to get this data from another source to avoid this limitation. 
@esloch, could it be downloaded from dataverse?

**Comment by esloch** [comment]
> ```
> Run git clone https://github.com/esloch/Data.git docker/dev_dumps
> Cloning into 'docker/dev_dumps'...
> Downloading alertademo_data.sql (612 MB)
> Error downloading object: alertademo_data.sql (41ff70b): Smudge error: Error downloading alertademo_data.sql (41ff70bf7725aead03246d7d24d05334a7a463c6e546d360571fd03b9ceeccbb): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
> 
> Errors logged to /home/runner/work/AlertaDengue/AlertaDengue/docker/dev_dumps/.git/lfs/logs/20201210T182041.069933696.log
> Use `git lfs logs last` to view the log.
> error: external filter 'git-lfs filter-process' failed
> fatal: alertademo_data.sql: smudge filter lfs failed
> warning: Clone succeeded, but checkout failed.
> You can inspect what was checked out with 'git status'
> and retry with 'git restore --source=HEAD :/'
> ```
> 
> it would be good to get this data from another source to avoid this limitation.
> @esloch, could it be downloaded from dataverse?

Sim @xmnlab, os arquivos da base de dados demo (dengue.sql.gz e infodengue.sql.gz são baixados do dataverse.

File: AlertaDengue/dados/views.py [hunk 1]
```diff
@@ -390,7 +390,7 @@ def get_context_data(self, **kwargs):
         # municipios, geocodigos = list(mundict.values()), list(mundict.keys())
         # results[d] = dbdata.load_serie_cities(geocodigos, d)
         for d in diseases:
-            case_series[d] = dbdata.get_series_by_UF(d, 52)
+            case_series[d] = dbdata.get_series_by_UF(d, 51)
```
Comment (xmnlab) [inline AlertaDengue/dados/views.py:393]: I guess this error is not related to the error on CI ... 
you should keep your PR as short as possible.

**Comment by eltonn** [comment]
@fccoelho quando possível podes fazer a review?

**Comment by esloch** [comment]
Waiting for this merge to update the CI workflow in PR #380.

**Review by fccoelho — APPROVED** [review]
Seems ok to me.

**Pull request merged**